jujugui/static/gui/src/app/utils/relation-utils.js

```diff
@@ -533,45 +531,50 @@ YUI.add('relation-utils', function(Y) {
-    if (service1.get('subordinate')) {
-      subordinate = service1;
-      application = service2;
-    } else if (service2.get('subordinate')) {
-      subordinate = service2;
-      application = service1;
+    // If either of the applications is a subordinate then assign the
+    // variable appropriately.
+    if (application1.get('subordinate')) {
+      subordinate = application1;
+      application = application2;
+    } else if (application2.get('subordinate')) {
+      subordinate = application2;
+      application = application1;
     }
     if (subordinate) {
       const appSeries = application.get('series');
-      // If there selected subordinate series does not match the application we
-      // are relating it to then it must be a multi-series subordinate and we
-      // need to update the series to match.
+      // If the selected subordinate series does not match the application we
+      // are relating it to and it is on a 'container' scope then it must be
+      // a multi-series subordinate and we need to update the series to match.
       if (subordinate.get('series') !== appSeries) {
-        const existingRelations = RelationUtils.getRelationDataForService(
-          db, subordinate);
-        // If there is an existing relation then we don't want to update the
-        // relation otherwise it will not match the existing application.
-        if (existingRelations.length === 0) {
-          // Update the subordinate series to match the application. We know
-          // that the subordinate is a multi-series charm with a series that
-          // matches the application otherwise we would not have got matching
-          // endpoints.
-          subordinate.set('series', appSeries);
-        } else {
-          // If there is an existing relation then show an error and cancel
-          // adding the relation.
-          db.notifications.add({
-            title: 'Subordinate series does not match',
-            message: 'Subordinates can only be related to applications with ' +
-              'the same series. This subordinate already has a relation to an' +
-              ' application with a different series.',
-            level: 'error'
-          });
-          return;
+        // If the scope isn't container then the series doesn't matter.
+        if (match.scope === 'container') {
+          const existingRelations = RelationUtils.getRelationDataForService(
+            db, subordinate);
+          // If there is an existing relation then we don't want to update the
+          // relation otherwise it will not match the existing application.
+          if (existingRelations.length === 0) {
+            // Update the subordinate series to match the application. We know
+            // that the subordinate is a multi-series charm with a series that
+            // matches the application otherwise we would not have got matching
+            // endpoints.
+            subordinate.set('series', appSeries);
```

Comment:
If the subordinate is connected to multiple apps with different series, will this just update the series to the last connected app? Does that matter?